save_princess_challenge/lib/display_path.rb

```diff
@@ -1,15 +1,54 @@
 #!/bin/ruby
+class DisplayPath
+  def initialize; end
```

Comment:
Could change methods to class methods since there isn't much need for instantiation